### DISCUSSION TITLE
Where is `not` documented?

### DISCUSSION THREAD

**Discussion by Schweber** [nushell/nushell#13078] [Q&A]
I just read https://github.com/nushell/nushell/pull/12837 and learned that i can use `not`.

I wasn't aware of that and just looked through the documentation but couldn't find it. Where is `not` described?

It's not under [Other operators](https://www.nushell.sh/lang-guide/chapters/operators.html#other-operators)
```
    =~ - Regex Match / Contains
    !~ - Not Regex Match / Not Contains
    in - In / Contains (doesn't use regex)
    not-in - Not In / Not Contains (doesn't use regex)
    starts-with - Starts With
    ends-with - Ends With
    && - And
    and - And
    || - Or
    or - Or
```

**Comment by fdncred** [comment]
`not` is an operator. You can see the operators with `help operators`.

**Comment by ghost** [comment]
it is [in the book](https://www.nushell.sh/book/operators.html#operators), but not in the [language guide](https://www.nushell.sh/lang-guide/chapters/operators.html#arithmetic-operators). should probably be added there as well.

**Comment by Schweber** [comment]
Thanks!
